en/camera/mavlink_camera.md

```diff
@@ -0,0 +1,88 @@
+
+To connect PX4 to your camera you need to attach the camera to an unused serial port on your flight controller, such as `TELEM2`, or even the Ethernet port (if there is one).
+If the camera is abstracted by a camera manager on a companion computer you would connect the companion computer to the unused serial port.
+
+Then you configure the port you selected as a [MAVLink Peripheral](../peripherals/mavlink_peripherals.md).
+The document explains how, but in summary:
+
+1. Modify an unused `MAV_n_CONFIG` parameter, such as [MAV_2_CONFIG](../advanced_config/parameter_reference.md#MAV_2_CONFIG), so that it is assigned to port to which you connected the camera/companion computer.
+1. Set the corresponding [MAV_2_MODE](../advanced_config/parameter_reference.md#MAV_2_MODE) to `2` (Onboard).
+   This ensures that the right set of MAVLink messages are emitted for a companion computer (or camera).
+1. Set [MAV_2_FORWARD](../advanced_config/parameter_reference.md#MAV_2_FORWARD) to enable forwarding of communications from the port to other ports, such as the one that is connected to the ground station.
+1. You may need to set some of the other parameters, depending on your connection.
+
+That's all for PX4.
+
+Depending on the camera/camera manager you may also need to set up some configuration parameters.
+For more information see:
+
+- [Using a Companion Computer with Pixhawk Controllers](../companion_computer/pixhawk_companion.md)
+- [Companion Computers > Companion Computer Software](http://localhost:5173/px4_user_guide/en/companion_computer/index.md#companion-computer-software): In particular note [MAVLink-Router](https://github.com/mavlink-router/mavlink-router), which you can setup to route MAVLink traffic between a serial port and an IP link (or other camera manager interface).
+- [SIYI A8 mini camera manager](https://github.com/julianoes/siyi-a8-mini-camera-manager) has a tutorial showing a setup with an RPi.
+
+## Camera Managers
+
+If you want to use a camera that does not natively support the MAVLink camera protocol you may be able to fine or write a MAVLink Camera manager.
+The camera manager runs on a companion computer and bridges between the MAVLink camera protocol interface and the camera's native interface.
+
+Generic/extensible camera managers:
+
+- [MAVLink Camera Manager](https://github.com/mavlink/mavlink-camera-manager) - Extensible cross-platform MAVLink Camera Server built on top of GStreamer and Rust-MAVLink.
+- [Dronecode Camera Manager](https://camera-manager.dronecode.org/en/) - Adds Camera Protocol interface for cameras connected to Linux computer.
+
+Camera-specfic camera managers:
+
+- [SIYI A8 mini camera manager](https://github.com/julianoes/siyi-a8-mini-camera-manager) - Community-supported MAVSDK-plugin based camera manager for the [SIYI A8 mini](https://shop.siyi.biz/products/siyi-a8-mini) (includes tutorial).
```

Comment:
I probably wouldn't even write "community-supported" because that would imply it's supported when it might not be.

Reply:
OK. I guess you can be explicit about the support you provide on the site.